jsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I haven't used Visual Studio Code - how strict is the schema? Can you do something like this?
```json
{
  "purpose": "This file is used to support Microsoft Visual Studio Code users. See http://blogs.msdn.com/b/vscode/archive/2015/07/06/vs-code-es6.aspx for more details.",
  "compilerOptions": {
    "target": "ES6",
    "module": "commonjs"
  }
}
```

If that doesn't work, disregard the original comment. I'm just worried about the proliferation of configuration files in the root directory (.eslint*, .git*, .travis*, *.conf*.js, gradlew*, etc.) and would like to provide some sort of manifest explaining what the files are and maybe how they can be integrated with various tools - but that is a bigger problem.